src/gettext.h

```diff
@@ -51,10 +51,19 @@ extern wchar_t *utf8_to_wide_c(const char *str);
 // The returned string is allocated using new
 inline const wchar_t *wgettext(const char *str)
 {
-	return utf8_to_wide_c(gettext(str));
+#if USE_GETTEXT
```

Comment:
Unnecessary use of preprocessors here. `gettext()` will functionally be ignored if gettext is unavailable. (See L38)

Reply:
@SmallJoker 
Yes, the gettext function will be ignored if it is compiled without gettext, I known the use of:
```#define gettext(String) String```
But why check if the string is empty if it is not going to be translated? 
Without gettext when compiling would be left:   
```
    return str[0] ? utf8_to_wide_c(str) : utf8_to_wide_c("");
```
and
```
    return text.empty() ? text.c_str() : text.c_str();
```
In both cases they'd only waste time testing a character for nothing, it's a bit ugly, isn't it?

Reply:
It's about the readability. The time spent for a conditional jump is negligible in this case.

Reply:
Ok